src/cr/cube/measures/index.py

```diff
@@ -46,13 +46,30 @@ def _mr_index(self, weighted):
         selected = table[self.cube.ind_selected]
         non_selected = table[self.cube.ind_non_selected]
 
-        if self.cube.mr_dim_ind == 0 or self.cube.mr_dim_ind == (0, 1):
+        # mr by mr
+        if self.cube.mr_dim_ind == (0, 1) or self.cube.mr_dim_ind == (1, 2):
+            row_proportions = self.cube.proportions(axis=0,
```

Comment:
This is not `row_proportions`, but `col_proportions`, because you're calculating it with `axis=0`. At least it should be called that way (the variable). If it doesn't produce the expected value, than that's a bug and should be fixed...